homeassistant/components/light/rflink.py

```diff
@@ -0,0 +1,136 @@
+"""Support for Rflink lights.
+
+For more details about this platform, please refer to the documentation
+at https://home-assistant.io/components/light.rflink/
+
+"""
+import asyncio
+import logging
+
+from homeassistant.components import group
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+import homeassistant.components.rflink as rflink
+
+from . import DOMAIN
+
+DEPENDENCIES = ['rflink']
+
+_LOGGER = logging.getLogger(__name__)
+
+VALID_CONFIG_KEYS = [
+    'aliasses',
+    'name',
+]
+
+
+def entity_type_for_device_id(device_id):
+    """Return entity class for procotol of a given device_id."""
```

Comment:
Since you're writing async code, please indicate in the doc string if this method is safe to run in both async and non-async world. [like this](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/core.py#L368)